--- Cargo.toml.old	2015-02-15 14:00:02.613705211 +0100
+++ Cargo.toml	2015-02-15 14:00:07.309705073 +0100
@@ -27,27 +27,6 @@
 [dev-dependencies]
 clock_ticks = "*"
 
-[target.arm-linux-androideabi.dependencies.android_glue]
-git = "https://github.com/tomaka/android-rs-glue"
-
-[target.i686-apple-darwin.dependencies.cocoa]
-git = "https://github.com/servo/rust-cocoa"
-
-[target.x86_64-apple-darwin.dependencies.cocoa]
-git = "https://github.com/servo/rust-cocoa"
-
-[target.i686-apple-darwin.dependencies.core_graphics]
-git = "https://github.com/servo/rust-core-graphics"
-
-[target.x86_64-apple-darwin.dependencies.core_graphics]
-git = "https://github.com/servo/rust-core-graphics"
-
-[target.i686-apple-darwin.dependencies.core_foundation]
-git = "https://github.com/servo/rust-core-foundation"
-
-[target.x86_64-apple-darwin.dependencies.core_foundation]
-git = "https://github.com/servo/rust-core-foundation"
-
 [target.i686-pc-windows-gnu.dependencies]
 winapi = "0.1"
 gdi32-sys = "*"
